bundles/org.openhab.binding.onewire/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,4 +6,27 @@
 	<name>OneWire Binding</name>
 	<description>This is the binding for OneWire.</description>
 	<connection>local</connection>
+
+	<discovery-methods>
+		<discovery-method>
+			<service-type>mdns</service-type>
+			<mdns-service-type>_owserver._tcp.local.</mdns-service-type>
+			<match-properties>
+				<match-property>
+					<name>name</name>
+					<regex>OWFS (1-wire) Server</regex>
+				</match-property>
+			</match-properties>
+		</discovery-method>
```

Comment:
Can you put that in a separate PR so that it can be merged quickly.
Is there a benefit to match the name while the service seems already very specific ?

Reply:
Hm, I just tried to confirm, but a newly installed instance of owserver on my test system did not announce via mdns at all. I have removed this part for now.